ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,63 @@
+"""
+Module which contains Config class. This class is singleton.
+It exists to parse *.conf files and return dictionary,
+which contains configuration from those files. Every 15 minutes
+it returns new dictionary which contains updated configs.
+"""
+import os
+import time
+
+from ConfigParser import SafeConfigParser
+
+from ecomap.utils import get_logger
+from ecomap.utils import Singleton
+
+REFRESH_TIME = 900
+PASSWORD = 'password'
+CONFIG_PATH = os.path.join(os.environ['CONFROOT'], 'ecomap.conf')
+
+
+class Config(object):
+    """
+    Singleton class to get configs
+    """
+    __metaclass__ = Singleton
+
+    def __init__(self):
+        self.config = {}
+        self.update_time = 0
+        self.path = CONFIG_PATH
+        self.log = get_logger('config_parser')
+        self.log.info('Create instance of Config parser.')
+
+    def get_config(self):
+        """
+        Call parse method if need.
+        Returns:
+            dictionary, containing configs
+        """
+        if self.update_time + REFRESH_TIME < time.time():
+            self.log.info('Refresh configs')
+            self.update_time = time.time()
+            self._parse_confs()
+        self.log.info('Return configs.')
```

Comment:
You don't need  self.log.info('Return configs.') line here. You will logging 'Return configs.' every time when you will need some variables from config file. It is very expensive.